docs/details.rst

```diff
@@ -285,6 +285,55 @@ You can also override the default by setting the environment variable
 setting ``HYPOTHESIS_VERBOSITY_LEVEL=verbose`` will run all your tests printing
 intermediate results and errors.
 
+~~~~~~~~~~~~~~~~~
+Settings Profiles
+~~~~~~~~~~~~~~~~~
+
```

Comment:
The documentation should also mention that there's an argument you can use if you're using the pytest plugin.